internal/srv/app.go

```diff
@@ -173,13 +173,13 @@ func (s *Server) updateDeployment(ctx context.Context, lb *loadBalancer) error {
 
 	values, err := s.newHelmValues(lb)
 	if err != nil {
-		s.Logger.Errorw("unable to prepare chart values", "error", err, "loadBalancer", lb.loadBalancerID.String())
+		s.Logger.Warnw("unable to prepare chart values", "error", err, "loadBalancer", lb.loadBalancerID.String())
 		return err
 	}
 
 	client, err := s.newHelmClient(hash)
 	if err != nil {
-		s.Logger.Errorw("unable to initialize helm client", "error", err, "loadBalancer", lb.loadBalancerID.String())
+		s.Logger.Warnw("unable to initialize helm client", "error", err, "loadBalancer", lb.loadBalancerID.String())
```

Comment:
Generally my rule has been: if it returns an error we should log an error. What's the advantage of using Warn instead?

Reply:
agreed. I only use `warn` if it's something that doesn't negatively impact the expected flow, but something that we should keep an eye on and address later

Reply:
I was thinking that since all the errors in the helper functions bubble up to the top then we probably dont need to log them as errors otherwise we'll get essentially duplicated logs

Reply:
I was thinking that since all the errors in the helper functions bubble up to the top then we probably dont need to log them as errors otherwise we'll get essentially duplicated logs